udp.go

```diff
@@ -9,6 +9,24 @@ import (
 	"golang.org/x/net/ipv6"
 )
 
+// This is the required size of the OOB buffer to pass to ReadMsgUDP.
+var udpOOBSize int
+
+func init() {
```

Comment:
As `init`? Maybe slightly neater to just have it as a function?
~~~
var udpOOBSize = func() int {
 // ...
}
~~~